query/src/sessions/session_ref.rs

```diff
@@ -57,14 +56,15 @@ impl Drop for SessionRef {
 impl Session {
     pub fn destroy_session_ref(self: &Arc<Self>) {
         if self.ref_count.fetch_sub(1, Ordering::Release) == 1 {
-            std::sync::atomic::fence(Acquire);
+            std::sync::atomic::fence(Ordering::Acquire);
             tracing::debug!("Destroy session {}", self.id);
             self.session_mgr.destroy_session(&self.id);
             self.quit();
         }
     }
 
     pub fn increment_ref_count(self: &Arc<Self>) {
+        std::sync::atomic::fence(Ordering::Release);
         self.ref_count.fetch_add(1, Ordering::Relaxed);
```

Comment:
why not fetch_add(1, Ordering::Release)? q.q

Reply:
As the [doc](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicIsize.html#method.fetch_add) said: Using Release makes the load part Relaxed.